### ISSUE TITLE
[Accessibility]: Cannot select multiple lines in code view

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36635]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

On desktop, you can select multiple lines by clicking on one line, holding shift and clicking on another line. It isn't clear what the equivalent of holding shift is on mobile. Holding + dragging doesn't seem to work.

### Expected behavior

There should be some way to select multiple lines, perhaps pressing + dragging in the line number gutter.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by varungandhi-src** [comment]
Related problem for screen reader navigation:
- https://github.com/sourcegraph/sourcegraph/issues/36527

**Comment by felixfbecker** [comment]
Closing because WCAG doesn't require touch/mobile interactions to work, it just requires the website to work on higher zoom levels/narrow viewports.

**Comment by varungandhi-src** [comment]
Just to clarify, you're saying that not all functionality needs to work in a mobile context for WCAG? Does mobile not count as a "narrow viewport"? Or is the notion of "what ought to work" something other than "all pieces of core functionality"?
